js/exporter/excel_creator.js

```diff
@@ -69,16 +70,15 @@ var ExcelCreator = Class.inherit({
             this._maxIndex[1] = Number(cellIndex);
         }
 
-        while(true) {
```

Comment:
@jsdmitry @IgnatovDan @nightskylark 
Or just replace `while(true)` with `for(;;)` here without additional changes?🤨

Reply:
Change just to fit lint rules... Something like this?

        var isCellIndexFound = false;
        while(!isCellIndexFound) {
            charCode = 65 + ((cellIndex >= max) ? (cellIndex % max) : Math.ceil(cellIndex));
            sheetIndex = String.fromCharCode(charCode) + sheetIndex;
 
            if(cellIndex >= max) {
                cellIndex = Math.floor(cellIndex / max) - 1;
            } else {
                isCellIndexFound = true;
            }
        }

Reply:
Thank you @IgnatovDan! Done.